<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDialog</class>
 <widget class="QDialog" name="AccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>405</width>
    <height>160</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lab_name">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_id">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Account:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_num">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lab_balance">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Balance:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_money">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lab_user">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Group:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_nuser">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Ordinary User</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
